src/models/core/AnswersConfig.ts

```diff
@@ -36,4 +37,47 @@ export interface AnswersConfig {
   additionalQueryParams?: {
     [key: string]: string | number | boolean
   }
-}
\ No newline at end of file
+}
+
+
+/**
+ * Configuration options for {@link AnswersCore}, following {@link BaseAnswersConfig},
```

Comment:
Rather than saying 'following', would it be clearer to say: 'which includes the options from'? Same idea for `AnswersConfigWithToken`